packages/office-ui-fabric-react/src/components/Text/Text.styles.ts

```diff
@@ -12,12 +12,12 @@ export const TextStyles: ITextComponent['styles'] = (props: ITextProps, theme: I
       theme.fonts.medium,
       {
         display: block ? (as === 'td' ? 'table-cell' : 'block') : 'inline',
-        fontFamily: (variantObject && variantObject.fontFamily) || 'inherit',
-        fontSize: (variantObject && variantObject.fontSize) || 'inherit',
-        fontWeight: (variantObject && variantObject.fontWeight) || 'inherit',
-        color: (variantObject && variantObject.color) || 'inherit',
-        mozOsxFontSmoothing: variantObject && variantObject.MozOsxFontSmoothing,
-        webkitFontSmoothing: variantObject && variantObject.WebkitFontSmoothing
+        fontFamily: variantObject.fontFamily,
+        fontSize: variantObject.fontSize,
+        fontWeight: variantObject.fontWeight,
+        color: variantObject.color,
```

Comment:
Right now, I don't believe any of the default font styles include color. I wonder if this should instead default to a color from the theme.

Reply:
Not sure if @betrue-final-final would still be the right person to ask about this? Maybe @dzearing?

Reply:
Maybe, but I think having it like this will at least have the same effect as before since `undefined` colors will just inherit the colors of the parent.

Reply:
David is an engineer, and I’m a designer. I don’t think they are interchangeable perspectives. 

In any event, what are we suggesting?

Reply:
Of course! By tagging you and David, I more so meant that I'm not sure if this is an engineering or a design question.

Should the Text component inherit its styles from its parent? Or should it default to a semantic slot?

Reply:
@khmakoto good point!

Reply:
@natalieethell after speaking with @betrue-final-final we've leaned towards just letting the color be undefined and have the default inherited color so that we keep consistency before/after the PR instead of introducing the color here because, while it might be the right thing to do in the long term, it might muddle this PR with unintended things breaking.

Reply:
Does this mean the PR is ready-to-go as is?

Reply:
Yes, it should be, @natalieethell has already approved.